content/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -10,138 +10,133 @@ card:
-| TCP      | Inbound   | 10252      | kube-controller-manager | Self                      |
+| プロトコル | 通信の向き | ポート範囲  | 目的                    | 使用者                    |
+|-----------|------------|------------|-------------------------|---------------------------|
+| TCP       | Inbound    | 6443*      | Kubernetes API server   | All                       |
+| TCP       | Inbound    | 2379-2380  | etcd server client API  | kube-apiserver, etcd      |
+| TCP       | Inbound    | 10250      | Kubelet API             | Self, Control plane       |
+| TCP       | Inbound    | 10251      | kube-scheduler          | Self                      |
+| TCP       | Inbound    | 10252      | kube-controller-manager | Self                      |
 
 ### ワーカーノード
 
-| Protocol | Direction | Port Range  | Purpose               | Used By                 |
-|----------|-----------|-------------|-----------------------|-------------------------|
-| TCP      | Inbound   | 10250       | Kubelet API           | Self, Control plane     |
-| TCP      | Inbound   | 30000-32767 | NodePort Services**   | All                     |
+| プロトコル | 通信の向き | ポート範囲   | 目的                    | 使用者                  |
+|-----------|------------|-------------|-------------------------|-------------------------|
+| TCP       | Inbound    | 10250       | Kubelet API             | Self, Control plane     |
+| TCP       | Inbound    | 30000-32767 | NodePort Services**     | All                     |
 
-** Default port range for [NodePort Services](/docs/concepts/services-networking/service/).
+** [NodePort Services](/docs/concepts/services-networking/service/)のデフォルトのポートの範囲
 
-Any port numbers marked with * are overridable, so you will need to ensure any
-custom ports you provide are also open.
+\* の項目は書き換え可能です。
+そのため、あなたが指定したカスタムポートも開いていることを確認する必要があります。
 
-Although etcd ports are included in control-plane nodes, you can also host your own
-etcd cluster externally or on custom ports.
+etcdポートはコントロールプレーンノードに含まれていますが、
+独自のetcdクラスターを外部またはカスタムポートでホストすることもできます。
 
-The pod network plugin you use (see below) may also require certain ports to be
-open. Since this differs with each pod network plugin, please see the
-documentation for the plugins about what port(s) those need.
+使用するPod ネットワークプラグイン（以下を参照）のポートも開く必要があります。 
+これは各Pod ネットワークプラグインによって異なるため、必要なポートについては
+プラグインのドキュメントを参照してください。
 
 ## ランタイムのインストール
 
-Since v1.6.0, Kubernetes has enabled the use of CRI, Container Runtime Interface, by default.
+v1.6.0以降、KubernetesはデフォルトでCRI(Container Runtime Interface)の使用を有効にしています。
 
-Since v1.14.0, kubeadm will try to automatically detect the container runtime on Linux nodes
-by scanning through a list of well known domain sockets. The detectable runtimes and the
-socket paths, that are used, can be found in the table below.
+また、v1.14.0以降、kubeadmは既知のドメインソケットのリストをスキャンして、
+Linuxノード上のコンテナランタイムを自動的に検出しようとします。 
+検出可能なランタイムとソケットパスは、以下の表に記載されています。
 
 | Runtime    | Domain Socket                    |
```

Comment:
Can I just check that these are intentionally left as-is?

Reply:
It's my mistake.
fixed it.